cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitorTest.java

```diff
@@ -574,38 +579,30 @@ private void waitForStateUpdate(LoadMonitor loadMonitor) {
-      futureByBroker.put(0, completedFuture(logdirInfoBylogdir));
-
-      logdirInfoBylogdir =  new HashMap<>();
-      replicaInfoByPartition = new HashMap<>();
-      replicaInfoByPartition.put(T0P0, new DescribeLogDirsResponse.ReplicaInfo(0, 0, false));
-      replicaInfoByPartition.put(T0P1, new DescribeLogDirsResponse.ReplicaInfo(0, 0, false));
-      replicaInfoByPartition.put(T1P0, new DescribeLogDirsResponse.ReplicaInfo(0, 0, false));
-      logdirInfoBylogdir.put("/tmp/kafka-logs-1", new DescribeLogDirsResponse.LogDirInfo(Errors.NONE, replicaInfoByPartition));
-      logdirInfoBylogdir.put("/tmp/kafka-logs-2",
-                             new DescribeLogDirsResponse.LogDirInfo(Errors.NONE,
-                                                                    Collections.singletonMap(T1P1,
-                                                                                             new DescribeLogDirsResponse.ReplicaInfo(0, 0, false))));
-      futureByBroker.put(1, completedFuture(logdirInfoBylogdir));
-      return constructor.newInstance(futureByBroker);
-    } catch (InstantiationException | IllegalAccessException | InvocationTargetException | NoSuchMethodException e) {
-      // Let it go.
-    }
-    return null;
+  private Map<Integer, KafkaFuture<Map<String, DescribeLogDirsResponse.LogDirInfo>>> getDescribeLogDirsResultValues() {
+
+    Map<Integer, KafkaFuture<Map<String, DescribeLogDirsResponse.LogDirInfo>>> futureByBroker = new HashMap<>();
+    Map<String, DescribeLogDirsResponse.LogDirInfo> logdirInfoBylogdir =  new HashMap<>();
+    Map<TopicPartition, DescribeLogDirsResponse.ReplicaInfo> replicaInfoByPartition = new HashMap<>();
+    replicaInfoByPartition.put(T0P0, new DescribeLogDirsResponse.ReplicaInfo(0, 0, false));
+    replicaInfoByPartition.put(T0P1, new DescribeLogDirsResponse.ReplicaInfo(0, 0, false));
+    replicaInfoByPartition.put(T1P0, new DescribeLogDirsResponse.ReplicaInfo(0, 0, false));
+    replicaInfoByPartition.put(T1P1, new DescribeLogDirsResponse.ReplicaInfo(0, 0, false));
+    logdirInfoBylogdir.put("/tmp/kafka-logs", new DescribeLogDirsResponse.LogDirInfo(Errors.NONE, replicaInfoByPartition));
+    futureByBroker.put(0, completedFuture(logdirInfoBylogdir));
+
+    logdirInfoBylogdir =  new HashMap<>();
+    replicaInfoByPartition = new HashMap<>();
+    replicaInfoByPartition.put(T0P0, new DescribeLogDirsResponse.ReplicaInfo(0, 0, false));
+    replicaInfoByPartition.put(T0P1, new DescribeLogDirsResponse.ReplicaInfo(0, 0, false));
+    replicaInfoByPartition.put(T1P0, new DescribeLogDirsResponse.ReplicaInfo(0, 0, false));
+    logdirInfoBylogdir.put("/tmp/kafka-logs-1", new DescribeLogDirsResponse.LogDirInfo(Errors.NONE, replicaInfoByPartition));
+    logdirInfoBylogdir.put("/tmp/kafka-logs-2",
+            new DescribeLogDirsResponse.LogDirInfo(Errors.NONE,
+                    Collections.singletonMap(T1P1,
+                            new DescribeLogDirsResponse.ReplicaInfo(0, 0, false))));
```

Comment:
nit: Do we enforce the indentation style to the parenthesis?

Reply:
Didn't notice this during the cherry-pick -- the indentation should indeed be fixed.
Thanks for pointing out!